qownnotes/snapcraft.yaml

```diff
@@ -10,7 +10,7 @@ description: |
 
 apps:
   qownnotes:
-    command: desktop-launch usr/bin/QOwnNotes --snap
+    command: desktop-launch ${SNAP}/usr/bin/QOwnNotes --snap
```

Comment:
That's why I vote for `desktop-launch ${SNAP}/usr/bin/QOwnNotes --snap` because there also is an executable `/usr/bin/QOwnNotes` (also accessible via $PATH) that should not be executed.